Source/WebInspectorUI/UserInterface/Views/Variables.css

```diff
@@ -406,8 +406,19 @@ body:is(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) {
         --slider-track-background: hsl(0, 0%, 35%);
         --slider-track-box-shadow: inset 0 0 3px hsl(0, 0%, 20%);
         --slider-track-tick-background: hsl(0, 0%, 55%);
+
+        --high-contrast-background-color: hsl(0, 0%, 10%);
+        --high-contrast-border-color: hsl(0, 0%, 10%);
+        --high-contrast-text-color: hsl(0, 0%, 90%);
     }
 
+    @media (prefers-contrast: more) {
+        :root {
+            /*Increase contrast for better accessibility*/
+            --background-color: var(--high-contrast-background-color);
+            --border-color: var(--high-contrast-border-color);
+            --text-color: var(--high-contrast-text-color);
+        }
```

Comment:
I think this is missing a closing brace for the media query

Reply:
Fixed